### PR TITLE
Prepare for for 200.690.0

### LINKED ISSUE #15: Compat entries for next version
Do we agree that the new compat entry for FLINT_jll should be "^200.690.0"?

### PR DESCRIPTION
**Pull request by thofma**
I chose 0.4.0 as the next version and hope that this is OK with you.

Also closes #15.

### CODE REVIEW

File: deps/build_FLINT.v200.690.0.jl [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+products = [
+    LibraryProduct(prefix, ["libflint"], :libflint),
+]
+
+# Download binaries from hosted location
+bin_prefix = "https://github.com/JuliaBinaryWrappers/FLINT_jll.jl/releases/download/FLINT-v200.690.0+0"
+
+# Listing of files generated by BinaryBuilder:
+download_info = Dict(
+    Linux aarch64 {libc=glibc} => ("$bin_prefix/FLINT.v200.690.0.aarch64-linux-gnu.tar.gz", "4a8aa3d405223e7334e9a53e4dde37455edc1431551e8f12102f3cf0835e0f28"),
```
Comment (fingolfin) [inline deps/build_FLINT.v200.690.0.jl:15]: The keys are wrong here. I am guessing platforms are not serialized correctly? Anyway, we need to match what works in Julia 1.0-1.2...
```suggestion
    Linux(:aarch64, libc=:glibc) => ("$bin_prefix/FLINT.v200.690.0.aarch64-linux-gnu.tar.gz", "4a8aa3d405223e7334e9a53e4dde37455edc1431551e8f12102f3cf0835e0f28"),
```

Comment (thofma) [reply deps/build_FLINT.v200.690.0.jl:15]: Working on it.

**Comment by thofma** [comment]
Should be good to go.

**Review by fingolfin — APPROVED** [review]
Thank you!

There are build failures in the Julia 1.5 tests in the "Test reverse dependencies" -- but I think those are "normal" because Nemo etc. have not yet been updated. Correct?

**Comment by benlorenz** [comment]
The reverse dependencies seem to use a rather old nemo version (Nemo v0.17.0) that might not be compatible with julia 1.5. One from before the LoadFlint era.

**Review by benlorenz — APPROVED** [review]

**Comment by fingolfin** [comment]
@benlorenz I don't think we hardcode a Nemo version; I think the problem is that newer Nemo versions fix a LoadFlint version; so the reverse test picks the newest Nemo it can find which doesn't preclude LoadFlint 0.4.

So I indeed think these will be fixed only once a fixed Nemo is released.

**Pull request merged**